lib/index.rb

```diff
@@ -4,6 +4,8 @@
 require 'sinatra'
 require './lib/file_customer_gateway'
```

Comment:
Could we change these `require`s to `require relative`s?

Reply:
What does that do?

Reply:
They're similar. It just reduces the typed path name since the index file is in the same directory.